### ISSUE TITLE
Support ws package version 6 as peer dependency

### ISSUE THREAD

**Issue by curran** [Teamwork/websocket-json-stream#2]
Greetings,

I'm seeing a warning:

```
npm WARN @teamwork/websocket-json-stream@1.1.0 requires a peer of ws@^5.2.0 but none is installed. You must install peer dependencies yourself.
```

I have `ws` version `6.1.0` as a dependency.

I believe this warning appears because `websocket-json-stream` uses `^`, not `<`, to declare the semver range.

Proposal: change `^` to `<` in `websocket-json-stream` peer dependencies declaration.

Thoughts?

**Comment by gkubisa** [comment]
Thanks for reporting this @curran . I updated the `ws` peer dependency to `>=5.2.0 <7.0.0`. The upper limit is needed because I can't guarantee that `websocket-json-stream` will work with `ws@7.0.0` - although it most likely will. :-) I published the fix as `websocket-json-stream@1.1.1`.

**Comment by curran** [comment]
Excellent! Thank you @gkubisa .
